playground/database-do/__tests__/e2e.test.mts

```diff
@@ -11,26 +12,29 @@ import { expect } from "vitest";
 
 setupPlaygroundEnvironment(import.meta.url);
 
-testSDK("seeds the database and displays initial todos", async ({ page }) => {
-  const devServerControl = createDevServer();
+(SKIP_DEV_SERVER_TESTS ? testSDK.skip : testSDK)(
```

Comment:
IIRC we have something for this already:
```
RWSDK_SKIP_DEV=1 pnpm test:e2e
```

More in [contributing.md](https://github.com/redwoodjs/sdk/blob/02491080de32836fad8059e5655984ca4740a98e/CONTRIBUTING.md#skipping-tests)

Or did I miss something here?

Reply:
Aaah, or does `testSDK` not use it maybe?

Sorry, maybe I should ask - howcome this change was needed?